vgen/src/test/java/io/gapi/vgen/testdata/library_gapic.yaml

```diff
@@ -97,6 +99,7 @@ interfaces:
     retry_codes_name: non_idempotent
     retry_params_name: default
     request_object_method: true
+    field_pattern: { 'name':'shelf', 'other_shelf_name':'shelf' }
```

Comment:
This isn't following the standard yaml syntax. You'd want something like:
field_pattern:
- name: shelf
- other_shelf_name: shelf

Maybe a more clarifying name would be better for this section - 'field_pattern' could be interpreted in multiple ways. So, maybe 'field_name_patterns' would be clearer. Note that it at least needs to be plural since there can be multiple patterns.